Update dashboards header screen shots (#19585)
DataDog/documentation: content/en/dashboards/_index.md

#### Version history

> From a dashboard click on the settings icon then click the version history option to open the Version History side panel. You can preview, restore, or clone your dashboard's version history. For more information, see the [Version History guide][12].

Reviewer: ```suggestion
From a dashboard, click on the settings icon and then click the version history option to open the Version History side panel. You can preview, restore, or clone your dashboard's version history. For more information, see the [Version History guide][12].
```

## Dashboard details

From an individual dashboard, hover over the dashboard title to view and edit dashboard details. A popover panel opens.

{{< img src="dashboards/suggested_dashboards.png" alt="Suggested dashboards" >}}

Under the title, the avatar tells you who created the dashboard.

### Suggested dashboards

From an individual dashboard, Datadog offers suggestions for viewing related dashboards. These dashboards are recommended based on the user activity in your organization and how often users go from this dashboard to other existing dashboards. 

> To add suggested dashboards list you need to add `[[suggested_dashboards]]` inside the dashboard description. 

Reviewer: ```suggestion
To add a suggested dashboards list, add `[[suggested_dashboards]]` inside the dashboard description. 
```
